modules/antismash.nf

```diff
@@ -3,6 +3,8 @@ process ANTISMASH {
     tag "${cluster_name}"
 
     container 'quay.io/microbiome-informatics/antismash:7.1.0.1_2'
+    
+    errorStrategy = { task.attempt <= 2 ? 'retry' : 'finish' }
```

Comment:
is there a plan to update to 8.1? It exists in nf-core and ASA

Reply:
For sure. They unfortunately didn't release it in time for this version and soil generation. I will update in the next release.